lib/edge.ig.yada/src/edge/yada/ig.clj

```diff
@@ -26,6 +26,16 @@
     id
     (assoc :id id)))
 
+(defmethod ig/init-key ::external-directories
```

Comment:
Maybe directory-resource is better? Then it can rely on the existing known name.

Reply:
Isn't `resource` not ideal because the directory won't be on the classpath?

Reply:
Resource would refer to web resource in this context, same as in yada